packages/react-form-state/README.md

```diff
@@ -0,0 +1,947 @@
+# `@shopify/react-form-state`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-formwell.svg)](https://badge.fury.io/js/%40shopify%2Freact-formwell.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-formwell.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-formwell.svg)
+
+Manage react forms tersely and type-safe with no magic.
+
+## Another form library?
+
+Good forms in React are surprisingly tough. A good api needs to support complex usecases with large numbers of fields split up across multiple components and explicit validation. It also needs to be quick to get started with and work for the simplest of forms cleanly. In meeting these goals, a package needs to balance developer experience, flexibility, and performance.
```

Comment:
What are `Good` forms?

Reply:
I think the rest of this paragraph says that :p